code/script.js

```diff
@@ -1,51 +1,199 @@
 // Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById("input-wrapper")
+const nameInput = document.getElementById("name-input")
+const button = document.getElementById("button")
 
-// If you need any global variables that you can use across different functions, declare them here:
+//global variables 
+let questionNumber = 1
 
+//Sound effects from Pixabay
+const soundEffectBot = () => {
+  const sound = new Audio('short-woosh-109592.mp3');
+  sound.play()
+}
+const soundEffectUser = () => {
+  const sound = new Audio('short-success-sound-glockenspiel-treasure-video-game-6346.mp3');
+  sound.play()
+}
+
+const soundEffectFinal = () => {
+  const sound = new Audio('sport-rock-logo1-13776.mp3');
+  sound.play()
+}
+
+const botSay = (words) => {
+  soundEffectBot();
+  setTimeout(() => showMessage(words, "bot"), 300);
+}
+
+const userSay = (words) => {
+  soundEffectUser();
+  setTimeout(() => showMessage(words, "user"), 300);
+}
 
-// Declare your functions after this comment
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
   if (sender === 'user') {
+    console.log(message);
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/Usericon.png" alt="User" />  
       </section>
     `
+
     // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
   } else if (sender === 'bot') {
+    console.log(message);
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/Botdino.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
+  // This makes the chat scroll to the last message
   chat.scrollTop = chat.scrollHeight;
 }
 
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userSay(`I would say ${guess}`);
+    setTimeout(() => whatSize(message), 1500)
+  }
+  else if (questionNumber === 2) {
+    userSay(message)
+    setTimeout(() => ifMonster(message), 1500)
+  }
+  else if (questionNumber === 3) {
+    userSay(message)
+    console.log("right track")
+    setTimeout(() => finish(message), 2000)
+  }
+}
+
+
```

Comment:
Numbering the questions in the Chatbot is great to keep the code organized. Great use of if statements.